client/my-sites/email/email-providers-comparison/price/price-information.tsx

```diff
@@ -73,63 +61,139 @@ const FreeTrialPriceInformation = ( {
 		},
 		comment:
-			"%(firstRenewalPrice)s and %(standardPrice)s are formatted prices with the currency (e.g. '$5')",
+			"%(firstRenewalPrice)s and %(standardPrice)s are formatted prices with the currency (e.g. '$5'). %(productTerm)s is either month or year",
 	};
 
-	if ( isGoogleWorkspace( product ) || ! isTitanMonthlyProduct( product ) ) {
+	if ( shouldProrate ) {
 		return (
 			<div className="price-information__free-trial">
 				{ translate(
-					'Try free today - first renewal at %(firstRenewalPrice)s after trial, regular price %(standardPrice)s per year',
+					'Try free today - first renewal at %(firstRenewalPrice)s after trial, regular price %(standardPrice)s per %(productTerm)s',
 					translateArguments
 				) }
 			</div>
 		);
 	}
 
-	if ( isTitanMonthlyProduct( product ) ) {
+	return (
+		<div className="price-information__free-trial">
+			{ translate(
+				'Try free today - renews at %(standardPrice)s after trial',
+				translateArguments
+			) }
+		</div>
+	);
+};
+
+const IntroductoryOfferPriceInformation = ( { product }: { product: ProductListItem } ) => {
+	const currencyCode = useSelector( getCurrentUserCurrencyCode );
+
+	const introductoryOffer = product.introductory_offer;
+	const shouldProrate = introductoryOffer?.should_prorate_when_offer_ends ?? false;
+	const numberOfDiscountedRenewals = Number( introductoryOffer?.transition_after_renewal_count );
+
+	const translateArguments = {
+		args: {
+			firstRenewalPrice: getFirstRenewalPrice( product, currencyCode ?? '' ),
+			numberOfDiscountedRenewals,
+			productTerm: product.product_term,
+			standardPrice: formatCurrency( product.cost ?? 0, currencyCode ?? '', { stripZeros: true } ),
+		},
+		comment:
+			"%(firstRenewalPrice)s and %(standardPrice)s are formatted prices with the currency (e.g. '$5'). %(productTerm)s is either month or year",
+	};
+
+	if ( shouldProrate ) {
 		return (
 			<div className="price-information__free-trial">
 				{ translate(
-					'Try free today - renews at %(standardPrice)s after trial',
+					'Discounted today - first renewal at %(firstRenewalPrice)s after trial, regular price %(standardPrice)s per %(productTerm)s',
 					translateArguments
 				) }
 			</div>
 		);
 	}
 
-	return null;
+	if ( numberOfDiscountedRenewals > 0 ) {
+		if ( numberOfDiscountedRenewals === 1 ) {
+			return (
+				<div className="price-information__free-trial">
+					{ translate(
+						'Discounted today and the first renewal, regular price %(standardPrice)s per %(productTerm)s',
+						translateArguments
+					) }
+				</div>
+			);
+		}
+
+		return (
+			<div className="price-information__free-trial">
+				{ translate(
+					'Discounted today and the next %(numberOfDiscountedRenewals)d renewals, regular price %(standardPrice)s per %(productTerm)s',
+					translateArguments
+				) }
+			</div>
+		);
+	}
+
+	return (
+		<div className="price-information__free-trial">
+			{ translate(
+				'Discounted today - renews at %(standardPrice)s after trial',
```

Comment:
The messaging would have to be updated here? Users will be renewing at regular price after the discount period is over and not a trial?

<img width="1067" alt="Screen Shot 2022-08-16 at 3 22 49 PM" src="https://user-images.githubusercontent.com/104910361/184964747-acf8954b-bc64-41ae-9c6c-5a0187f31b1b.png">

I'm also a bit confused about this particular scenario here. I see that there's a 1 year discount (amount unspecified), then on top of it, we're applying an extra 20% off? Is that the idea?